transcom/mymove
# INT B-18543 deprecate createSITAddressUpdate request and remove tech debt

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18543)

## Summary

With the deprecation of the [createSITAddressUpdate](https://transcom.github.io/mymove-docs/api/prime/v1#tag/sitAddressUpdate) endpoint, we need to remove all relevant tech debt and update the Swagger documentation.

I have taken the liberty of painting the codebase red and rid the earth of any mention of `SITAddressUpdate` or `sit_address_update`.

This PR does the following:
- removed all handlers and service objects related to creating a SIT address update request
- removed all handlers and service objects related to approving, rejecting, or updating SIT address update requests
- dropping the `sit_address_updates` table as well as the related enum `sit_address_update_status`
- scoured the UI code and removed any unused variables and props used that is related to SIT address updates on the service item level

### How to test

1. Fire up `client` and `server`
2. Navigate the prime sim and GHC applications and make sure you don't get any errors
3. Specific areas would be the MTO page, reviewing service items, and address forms
4. Confirm you can't call the prime `POST` endpoint `/prime/v1/sit-address-updates`
